I think this linting error might prevent even this PR.

```
mypy.....................................................................Failed
- hook id: mypy
- exit code: 1

app/email/notifications.py:44: error: Expected keyword arguments, {...}, or dict(...) in TypedDict constructor
Found 1 error in 1 file (checked 68 source files)
```

Does anyone have any ideas as to how to resolve this? When looking it seems that the requirements the linter mentions here are already fulfilled in the notifications module.

--- a/app/email/notifications.py
+++ b/app/email/notifications.py
@@ -42,9 +42,9 @@ def notify_atbd_version_contributors(
     if (len(data['notify']) > 0):
         user_notifications = [
             UserNotification(
-                **app_users[user_sub].dict(),
+                **app_users[user_sub].dict(), # type: ignore
                 notification=notification,
-                data=data, # type: ignore
+                data=data
             )
             for user_sub in data['notify']
         ]

--- a/app/email/notifications.py
+++ b/app/email/notifications.py
@@ -44,7 +44,7 @@ def notify_atbd_version_contributors(
             UserNotification(
                 **app_users[user_sub].dict(),
                 notification=notification,
-                data=data,
+                data=data, # type: ignore
             )
             for user_sub in data['notify']
         ]

